packages/components/datagrid/src/js/datagrid.controller.js

```diff
@@ -360,8 +360,13 @@ export default class DatagridController {
     return this.refreshDatagridPromise;
   }
 
-  sort(column) {
-    if (!column || !column.sortable) {
+  sort(column, event) {
+    if (
+      !column
+      || !column.sortable
+      || event.target.classList.contains('oui-popover-button')
```

Comment:
Can we avoid multiple || here ?

![image](https://github.com/ovh/ovh-ui-kit/assets/8767496/ce7f5afe-256f-49a7-89b8-0f9422086298)